### PR TITLE
Degeneracy Hunter: minor bug fix

### PR DESCRIPTION
**Pull request by adowling2**
## Changes proposed in this PR:
- Fixes printing mistake in Degeneracy Hunter (indentation error)

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by adowling2** [comment]
This is needed for https://github.com/IDAES/examples-pse/pull/21

**Review by andrewlee94 — DISMISSED** [review]
A couple of things to think about for the future (don't need to hold up this PR):

1. Should `print_level` use an `Enum` or similar to help guide users towards valid inputs. Alternatively, do you have a check elsewhere to screen for invalid inputs?
2. Rather than a simple print statement, it would be good to have some control over the output destination (e.g. a file or some other output channel), but defaulting to `stdout` if no input is provided.

File: idaes/core/util/model_diagnostics.py [hunk 1]
```diff
@@ -145,20 +145,20 @@ def check_residuals(self, tol=1e-5, print_level=2, sort=True):
...
+                        print("\ncount =",i,"\t|residual| =",r)
+                        c.pprint()
+        
+                    if print_level == 2:
+                        # print values and bounds for each variable in the constraint
+                        print("variable\tlower\tvalue\tupper")
+                        for v in identify_variables(c.body):
+                            self.print_variable_bounds(v)
             else:
                 print("No constraints with residuals larger than",tol,"!")
```
Comment (blnicho) [inline idaes/core/util/model_diagnostics.py:163]: @adowling2 you might want to double check this change. If I'm following your code I think the indentation on the for loop was correct and the problem is with the `else` statement on line 162. I think that needs to line up with the `if` on line 140. With this change nothing will be printed if `sort=False` which I don't think is what you intended.

Comment (ksbeattie) [reply idaes/core/util/model_diagnostics.py:163]: I'd think a unit test for this would help here?

**Comment by adowling2** [comment]
@blnicho Thank you for catching this. I fixed the indent. I still need to test this on the example.

**Comment by adowling2** [comment]
I verified the example runs correctly with this fix. @blnicho @andrewlee94 

@lbianchi-lbl This is ready to merge with the appropriate reviews.

**Review by jsiirola — APPROVED** [review]

**Review by andrewlee94 — APPROVED** [review]
Looks good to me.

**Pull request merged**